spec/spec_helper.rb

```diff
@@ -0,0 +1,20 @@
+require "log_decorator"
+
+def log_init
+  require "logger"
+
+  log = Logger.new(STDERR)
+  log.formatter = lambda { |_severity, _datetime, _progname, msg| "#{msg}\n" }
+  log.level = case ENV['LOG_LEVEL']
+              when 'ERROR' then Logger::ERROR
+              when 'WARN'  then Logger::WARN
+              when 'INFO'  then Logger::INFO
+              when 'DEBUG' then Logger::DEBUG
+              else              Logger::OFF
```

Comment:
@movitto I converted this from Log4r to Logger, to eliminate the Log4r dependancy. I guess Logger doesn't have an "OFF" level. I'll change the default to "ERROR".